src/backend/web/routes/posts.js

```diff
@@ -80,4 +81,27 @@ posts.get('/count', async (req, res) => {
   }
 });
 
+// Allow for the id to be a URI, possibly containing path separators
+posts.get('/*', async (req, res) => {
```

Comment:
Let me ask you a silly question: this won't confuse express?  That is `posts/` and `posts/....` are different enough?  Because we use `*` here vs. a named arg, in theory nothing (i.e., passing nothing after `/`) would match, no?

I feel like it must not be a problem, or your tests would be failing, but can you confirm and educate me?

Reply:
I actually had no idea, so I did some research. according to this source: https://stackoverflow.com/questions/6161567/express-js-wildcard-routing-to-cover-everything-under-and-including-a-path
> If you look at line 331 of the connect router the * in a path is replaced with .+ so will match 1 or more characters

however, wildcard routing seems to be not recommended. Therefore, I've changed it to `posts.get('/:guid')`.
also from the same source:
> For those who are learning node/express (just like me): do not use wildcard routing if possible!
from another source: https://blog.praveen.science/wildcard-routing-is-an-anti-pattern/